sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -62,6 +65,43 @@
 SUPPORTED_TEMPEST_CONFIG = set(["schedule"])
 
 
+@dataclass
```

Comment:
Can you check if you can use Pydantic base model instead of dataclass unless you see an advantage of using dataclass in this scenario.